.github/workflows/atfe_nightly_build_and_test.yml

```diff
@@ -29,27 +30,54 @@ jobs:
       fail-fast: false  # Prevents one job failure from cancelling all
       matrix:
         include:
+          # build.sh
           - build_script: build.sh
             test_script: test.sh
+            target_os: Ubuntu-22.04-x86_64
             runner: ToolchainGroup_LargeRunner_Linux-x64
+
+          - build_script: build.sh
+            test_script: test.sh
+            target_os: Ubuntu-22.04-arm64
+            runner: ah-ubuntu_22_04-c7g_8x-100
+
+          - build_script: build.ps1
+            test_script: test.ps1
+            target_os: Windows-2022-x86_64
+            runner: ToolchainGroup_LargeRunner_Windows-x64
+
+          # build_llvmlibc_overlay.sh
```

Comment:
Ditto - can we replace this with something like `LLVM-libc Overlay Package Build`?